recipe_scrapers/ingoodflavor.py

```diff
@@ -0,0 +1,18 @@
+# mypy: allow-untyped-defs
```

Comment:
@jknndy these `allow-untyped-defs` directives can be removed now (credit: #1174)